### ISSUE TITLE
BIBOX change API URL

### ISSUE THREAD

**Issue by fbadiola** [ccxt/ccxt#7988]
- OS: Linux
- Programming Language version: JS (Node.js)
- CCXT version: 1.37.70

Hi!

Bibox have problems with their `bibox.com` domain, [they explain it on twitter](https://twitter.com/Bibox365/status/1327078873813585920).

They change their api urls to:
* https://api.bibox365.com/v3/mdata/ping
* https://api.bibox.me/v3/mdata/ping

They didn't update their http documentation but they did on [websocket documentation](https://biboxcom.github.io/v3/spot/en/#websocket) where there's a reference to `wss://push.bibox.me`.

We've a lot of problems by this like you could imagine.

Thanks!

**Comment by kroitor** [comment]
@fbadiola thanks so much for reporting! 
Since at this time it is not clear which of the domains will be the default, we will keep watching for a short while.
If you wish to quickfix it with the existing version of CCXT, just override the API URL upon instantiation:

```
ccxt.bibox({
    'enableRateLimit': True,
    'urls': {
        'api': 'https://api.bibox365.com',
    },
})
```

**Comment by kroitor** [comment]
Fixed this issue in 1.37.71, the new build will arrive in 20 minutes. The hostname will default to `bibox365.com` for now, you can also override it like so:

```Python
ccxt.bibox({
    'enableRateLimit': True,
    'hostname': 'bibox.me',  # or bibox365.com or bibox666.com
})
```

**Comment by fbadiola** [comment]
Thanks for the quickness!!
